catalog/spatial/csw/spatial-csw-endpoint/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/CswEndpoint.java

```diff
@@ -661,30 +670,34 @@ private int deleteRecords(DeleteAction deleteAction)
 
         queryRequest = queryFactory.updateQueryRequestTags(queryRequest,
                 schemaTransformerManager.getTransformerSchemaForId(deleteAction.getTypeName()));
-
-        QueryResponse response = framework.query(queryRequest);
-
-        List<String> ids = new ArrayList<>();
-
-        for (Result result : response.getResults()) {
-            if (result != null && result.getMetacard() != null) {
-                ids.add(result.getMetacard()
-                        .getId());
+        queryRequest = setQueryPageSize(queryRequest);
```

Comment:
❓ Deleting metacards 20 at a time seems a little sub-optimal. Should we either increase `DEFAULT_PAGE_SIZE` or provide the page size parameter to the `setQueryPageSIze()` and give a higher number here?